libpal/wrmsr_x64.asm

```diff
@@ -0,0 +1,15 @@
+.code
+
+pal_execute_wrmsr proc
+    ; The address is already loaded in rcx because
+    ; it is the first integer argument. The value to
+    ; write is in rdx. We need that to be in edx:eax.
+    mov rax, rdx;
+    shr rdx, 032h;
```

Comment:
I think this one got missed (032h vs. 032)

Reply:
Fixed!